ingests/ihme_child_mortality.py

```diff
@@ -65,5 +63,33 @@ def get_location_hierachies():
     return locations
 
 
+def get_number_and_rates(start_year: int, end_year: int, temp_dir: str, locations: List[str]) -> pd.DataFrame:
+    """
+    Download the data on the number and rate of child mortality.
+    """
+    url = f"https://ghdx.healthdata.org/sites/default/files/record-attached-files/IHME_GBD_2019_U5M_{start_year}_{end_year}_CT_RT_0.zip"
+    r = requests.get(url)
+    z = zipfile.ZipFile(io.BytesIO(r.content))
+    z.extractall(temp_dir)
+    csv_file = f"IHME_GBD_2019_U5M_{start_year}_{end_year}_CT_RT_Y2021M09D01.CSV"
```

Comment:
If these paths are going to change consistently next time you run the script (for the next version), it would be convenient to define these file names with place holders, like `f"IHME_GBD_{GBD_YEAR}_U5M_{start_year}_{end_year}_CT_RT_Y{SOME_OTHER_YEAR}M{SOME_MONTH}D01.CSV"` (whatever the underlying logic is) and define those global variables at the top. With this, on the next update you'll just need to duplicate the current script, and change those variables. But if you don't think they are consistent on these namings, then it's probably not worth it.

Reply:
Good idea but I think they are unlikely to be consistent - the structure is pretty different from the previous update!